lib/brewdler/utils.rb

```diff
@@ -8,11 +8,28 @@ def self.system cmd, *args
     $?.success?
   end
 
-  def self.brew_installed?
-    system("brew --version &>/dev/null")
+  def brew_installed?
+    Kernel.system("brew --version &>/dev/null")
   end
 
-  def self.cask_installed?
-    system("brew cask --version &>/dev/null")
+  def cask_installed?
+    Kernel.system("brew cask --version &>/dev/null")
+  end
+
+  def brewfile
+    File.read(Dir['{*,.*}{B,b}rewfile'].first.to_s)
+  rescue Errno::ENOENT
+    raise "No Brewfile found"
+  end
+
+  def getc  # NOTE only tested on OS X
+    Kernel.system "/bin/stty raw -echo"
+    if STDIN.respond_to?(:getbyte)
+      STDIN.getbyte
+    else
+      STDIN.getc
+    end
+  ensure
+    Kernel.system "/bin/stty -raw echo"
   end
```

Comment:
I think `cleanup` is a kind of dangerous action. So I would like to ask for confirmation to users. What if you choose the wrong Brewfile? 

However, you can always make it non-interactive using `--force` option.